session.go

```diff
@@ -2111,14 +2111,15 @@ func NewTraceWriter(session *Session, w io.Writer) Tracer {
 
 func (t *traceWriter) Trace(traceId []byte) {
 	var (
-		coordinator string
-		duration    int
+		coordinator     string
+		duration        int
+		coordinatorPort int
 	)
-	iter := t.session.control.query(`SELECT coordinator, duration
-			FROM system_traces.sessions
-			WHERE session_id = ?`, traceId)
+	iter := t.session.control.query(`SELECT coordinator, duration, coordinator_port
```

Comment:
We can switch places of duration and coordinator_port fields a little bit:
```
SELECT coordinator, coordinator_port, duration
```

It will be more logical to display info about coordinator node this way